### PR TITLE
[keyboard] Plain60 cleanups

### PR DESCRIPTION
**Pull request by yanfali**
Discord user typischt requested access to bootmagic and a reset coed on default keymap.

## Description
  - enable bootmagic lite to make resetting easier
  - add a RESET keycode to default keymap on a layer
    - override KC_UP and KC_APP to be MO(1)
  - remove obsolete comments about sizes from rules.mk

This seem ok @Maartenwut ?

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/maartenwut/plain60/keymaps/default/keymap.c [hunk 1]
```diff
@@ -5,13 +5,21 @@
...
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
 [_MA] = LAYOUT(
   KC_ESC,  KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSLS, KC_BSPC, \
   KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          \
   KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT, KC_BSLS, KC_ENT,           \
-  KC_LSFT, KC_NUBS, KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,            \
-  KC_LCTL, KC_LGUI, KC_LALT,                        KC_SPC,                          KC_RALT, KC_RGUI, KC_APP,  KC_RCTRL)
+  KC_LSFT, KC_NUBS, KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, MO(_FN),            \
+  KC_LCTL, KC_LGUI, KC_LALT,                        KC_SPC,                          KC_RALT, KC_RGUI, MO(_FN),  KC_RCTRL),
```
Comment (zvecr) [inline keyboards/maartenwut/plain60/keymaps/default/keymap.c]: Would `LT(1,KC_APP)` be better here, so it still kinda maintains the previous behaviour on windows but gives access to the newly added layer?

Comment (yanfali) [reply keyboards/maartenwut/plain60/keymaps/default/keymap.c]: good call

File: keyboards/maartenwut/plain60/keymaps/default/keymap.c [hunk 2]
```diff
@@ -5,13 +5,21 @@
 // Layer names don't all need to be of the same length, obviously, and you can also skip them
 // entirely and just use numbers.
 #define _MA 0
+#define _FN 0
```
Comment (zvecr) [inline keyboards/maartenwut/plain60/keymaps/default/keymap.c]:
```suggestion
#define _FN 1
```

or the whole enum thing maybe?

File: keyboards/maartenwut/plain60/keymaps/default/keymap.c (hunk 1 above)
Comment (zvecr) [reply keyboards/maartenwut/plain60/keymaps/default/keymap.c]: As for KC_UP, how likely is a user going to hold down the key?

File: keyboards/maartenwut/plain60/keymaps/default/keymap.c (hunk 2 above)
Comment (yanfali) [reply keyboards/maartenwut/plain60/keymaps/default/keymap.c]: good catch

File: keyboards/maartenwut/plain60/keymaps/default/keymap.c (hunk 1 above)
Comment (yanfali) [reply keyboards/maartenwut/plain60/keymaps/default/keymap.c]: well that's the compromise of LT. So they'll have to tap it. I mean honestly that's a poor key to put in that location imo, so I'm just leaving it be.

Comment (yanfali) [reply keyboards/maartenwut/plain60/keymaps/default/keymap.c]: I believe, it works like this. Tap it twice. Second tap is a hold it repeats.

**Pull request merged**